web-app/src/components/TeamOverview/container.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import { PropTypes as PT } from 'prop-types';
+import Swal from 'sweetalert2';
+import { getTeamOverview } from '../../services/dashboardService';
+
+const TeamOverviewContainer = Wrapped =>
+  class extends React.Component {
+    static propTypes = {
+      history: PT.object.isRequired,
+    };
+
+    constructor(props) {
+      super(props);
+      this.state = {
+        teams: [],
+        selectedUser: {},
+      };
+    }
+
+    componentDidMount() {
+      
+      var date = new Date().toISOString().slice(0,10);
```

Comment:
Try to use moment dates. I can go over what you should do here instead to follow convention.

Reply:
I know you'll be changing this but try to use const, let etc.

Reply:
yeah no problem,....thought I had it as const, sorry about that. : )

Reply:
Easy done :)